samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala

```diff
@@ -129,9 +129,6 @@ class KafkaCheckpointManager(
     if (taskNamesToOffsets == null) {
       info("No TaskName to checkpoint mapping provided.  Reading for first time.")
       taskNamesToOffsets = readCheckpointsFromLog()
-    } else {
```

Comment:
Why delete this?

Reply:
there is no need for second read.
But for now this change is deleted from this PR.